pkg/kiln/io.go

```diff
@@ -295,3 +300,22 @@ func GetExampleDockerfile(dockerInfo *DockerInfo) (buf *bytes.Buffer, err error)
 
 	return buf, nil
 }
+
+// DetermineBaseImage consumes a given runtime selection and determines the base image for the Dockerfile
+func DetermineBaseImage(runtimeSelection string) (baseImage string, err error) {
+	runtimeSplit := strings.Split(runtimeSelection, ":")
+
+	switch runtimeSplit[0] {
```

Comment:
Maybe we could turn this into a `RuntimeMap` or something where we map `runtime` values to a known base image?